README.md

```diff
@@ -619,3 +620,10 @@ See [contributing.md](CONTRIBUTING.md) for sgd contribution principles.
 ## License
 
 This project license is MIT - see the [LICENSE.md](LICENSE.md) file for details
+
+## Complimentary Plugins
+
+These plugins have been designed to work with SGD:
+
+- [apex-test-list](https://github.com/renatoliveira/apex-test-list) - Developer: [renatoliveira](https://github.com/renatoliveira) - This plugin determines the specified Apex tests by reading test annotations made anywhere inside your Apex classes. You can have this plugin scan the package.xml created by SGD to determine the required Apex tests to run during deployment.
+- [apex-tests-git-delta](https://github.com/mcarvin8/apex-tests-git-delta) - Developer: [mcarvin8](https://github.com/mcarvin8) - This plugin determines the specified Apex tests by reading the commit messages in the commit range. You can use the same `--from` and `--to` commit hashes when using SGD and apex-tests-git-delta to determine the required Apex tests to run during deployment.
```

Comment:
**suggestions:** could it be moved just before the `Changelog` section please ?

Reply:
moved before changelog